### PR TITLE
OffSeason2022: Renamed timer in ClimberTimer to m_timer

### PR DESCRIPTION
**Pull request by hpark-24**

### CODE REVIEW

File: Swerve2022/src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 1]
```diff
@@ -38,6 +38,8 @@ public class Drivetrain extends SubsystemBase
   private final Translation2d         m_backLeftLocation   = new Translation2d(-0.381, 0.381);
   private final Translation2d         m_backRightLocation  = new Translation2d(-0.381, -0.381);
 
+  private final String                kcanbusstring        = DTConsts.kcanbusstring;
```
Comment (jmullins16) [inline Swerve2022/src/main/java/frc/robot/subsystems/Drivetrain.java:41]: Not sure why we need a member variable to hold this constant value. Just use the constant wherever needed.

**Review by jmullins16 — CHANGES_REQUESTED** [review]
Minor changes, but it all looks functional and correct.

**Review by jmullins16 — APPROVED** [review]
Looks great!

**Review by reevar24 — APPROVED** [review]
Looks good!

**Pull request merged**